modules/exploits/multi/http/wp_suretriggers_auth_bypass.rb

```diff
@@ -17,26 +18,30 @@ def initialize(info = {})
     super(
       update_info(
         info,
-        'Name' => 'WordPress SureTriggers Auth Bypass and RCE',
+        'Name' => 'WordPress SureTriggers/OttoKit Combined Auth Bypass (CVE-2025-3102, CVE-2025-27007)',
```

Comment:
Could we make it more clear to the user that OttoKit is the same thing as SureTriggers, as there's two CVEs being exploited here. 
```suggestion
        'Name' => 'WordPress SureTriggers (aka OttoKit) Combined Auth Bypass (CVE-2025-3102, CVE-2025-27007)',
```